vault/dendron.topic.pod.builtin.github-issue.publish.md

```diff
@@ -78,12 +78,39 @@ This is the discussion for [RFC 28 - Notifications](https://wiki.dendron.so/note
 
 If set to false, the default body for Discussion would be : Discussion for `<title of the note>`
 
+### aliasMapping
+- description : aliases to the frontmatter fields to support task note mappings.
+- type: object
+- required: false
+
+Github issues make it possible to provide aliases to the frontmatter fields so that it can support task note mappings. It allows us to directly use a GitHub issue as a task note and then publish the results back to GitHub when we update the task status.
```

Comment:
```suggestion
GitHub issues make it possible to provide aliases to the frontmatter fields so that it can support task note mappings. It allows us to directly use a GitHub issue as a task note and then publish the results back to GitHub when we update the task status.
```